docs/core/tools/global-tools-how-to-create.md

```diff
@@ -0,0 +1,201 @@
+             .'....'..'..........'..'.......'.
+             .'..................'...   ......
+             .  ......'.........         .....
+             .    _            __        ......
+            ..    #            ##        ......
+           ....       .                 .......
+           ......  .......          ............
+            ................  ......................
+            ........................'................
+           ......................'..'......    .......
+        .........................'..'.....       .......
+     ........    ..'.............'..'....      ..........
+   ..'..'...      ...............'.......      ..........
+  ...'......     ...... ..........  ......         .......
+ ...........   .......              ........        ......
+.......        '...'.'.              '.'.'.'         ....
+.......       .....'..               ..'.....
+   ..       ..........               ..'........
+          ............               ..............
+         .............               '..............
+        ...........'..              .'.'............
+       ...............              .'.'.............
+      .............'..               ..'..'...........
+      ...............                 .'..............
+       .........                        ..............
+        .....
+";
+    Console.WriteLine(bot);
+}
+```
+
+### Test the tool
+
+You can run the project and see the output. Try these variations to run our application:
+
+```csharp
+dotnet run
+dotnet run -- "Hello from the bot"
+dotnet run -- hello from the bot
+```
+
+All arguments after the `--` delimiter are passed to our application.
+
+## Setup the global tool
+
+Before you can pack and distribute the application as a Global Tool, you need to modify the project file. Open the `botsay.csproj` file and add three new XML nodes to the `<Project><PropertyGroup>` node:
+
+- \<PackAsTool>  
+Indicates that the application will be packaged for install as a Global Tool.
+
+- \<ToolCommandName>  
+An alternitive name for the tool, otherwise the tool executable will be named after the project file.
+
+- \<PackageOutputPath>  
+Where the NuGet package will be produced. The NuGet package is what the .NET Core CLI Global Tools uses to install your tool.
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+
+    <PackAsTool>true</PackAsTool>
+    <PackageOutputPath>./nupkg</PackageOutputPath>
+    <ToolCommandName>botsay</ToolCommandName>
+
+  </PropertyGroup>
+
+</Project>
+```
+
+Next, create a NuGet package for your application.
+
+```console
+dotnet pack -c Release
+```
+
+The `botsay.1.0.0.nupkg` file will be created in the folder identified by the `<PackageOutputPath>` XML value from the `botsay.csproj` file, which should be the `./nupkg` folder.
+
+Now that you have a package, install the tool from that package: 
```

Comment:
Consider clarifying that this is for testing and you'll later put it on a feed and that once on a feed the user is using, they won't have to do the --add-source part